migrations/versions/1060_add_external_id_column.py

```diff
@@ -0,0 +1,77 @@
+"""Add external ID column
+
+Revision ID: 1050
+Revises: 1040
+Create Date: 2017-11-23 16:46:00.569239
+
+"""
+from alembic import op
+import json
+import random
+import sqlalchemy as sa
+from sqlalchemy.sql import text
+
+# revision identifiers, used by Alembic.
+revision = '1060'
+down_revision = '1050'
+
+DIRECT_AWARD_AUDIT_TYPES = ('create_project', 'create_project_search', 'lock_project', 'downloaded_project')
+
+
+def random_positive_external_id():
+    return random.SystemRandom().randint(10 ** 14, (10 ** 15) - 1)
+
+
+def upgrade():
+    op.add_column('direct_award_projects', sa.Column('external_id', sa.BIGINT()))
```

Comment:
`supplier_id` was defined as `sa.BigInteger()` - I have no idea if there's any difference between these things, but maybe use that here for consistency?